doc: applications: SLM adding links to AT manual (#4476)
nrfconnect/sdk-nrf: applications/serial_lte_modem/README.rst

#########################

The Serial LTE Modem (SLM) application can be used to emulate a stand-alone LTE modem on the nRF9160.

> The application accepts both the modem-specific AT commands documented in the `nRF91 AT Commands Reference Guide <AT Commands Reference Guide_>`_ and proprietary AT commands that run on the application processor.

Reviewer: ```suggestion
The application accepts both the modem-specific AT commands documented in the `nRF91 AT Commands Reference Guide <AT Commands Reference Guide_>`_ and proprietary AT commands that is documented in "https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/applications/serial_lte_modem/doc/AT_commands_intro.html" (nice link text)
```
Another suggestion is that we change the title (from the linked item) "AT commands" to "SLM AT commands" to easier differentiate between modem specific and SLM ?

Author: Updated.
